src/__tests__/views/CacheConfiguration.test.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+// import { useFetchConfiguration } from '@app/services/configHook';
+import * as ConfigHook from '@app/services/configHook';
+
+import { CacheConfiguration } from '@app/Caches/Configuration/CacheConfiguration';
+import { renderWithRouter } from '../../test-utils';
+
+jest.mock('@app/services/configHook');
+const mockedConfigHook = ConfigHook as jest.Mocked<typeof ConfigHook>;
+
+const jsonConfig = `{
+  "distributed-cache": {
+    "mode": "SYNC",
+    "statistics": true,
+    "encoding": {
+      "key": {
+        "media-type": "application/x-protostream"
+      },
+      "value": {
+        "media-type": "application/x-protostream"
+      }
+    }
+  }
+}`;
+const yamlConfig = `distributedCache: 
+mode: "SYNC"
+statistics: "true"
+encoding: 
+  key: 
+    mediaType: "application/x-protostream"
+  value: 
+    mediaType: "application/x-protostream"`;
+
+const xmlConfig = `<?xml version="1.0"?>
+<distributed-cache mode="SYNC" statistics="true">
+	<encoding>
+		<key media-type="application/x-protostream"/>
+		<value media-type="application/x-protostream"/>
+	</encoding>
+</distributed-cache>`;
+
+describe('Configuration Page', () => {
+  test('YAML Configuration testing', () => {
+    mockedConfigHook.useFetchConfigurationYAML.mockImplementationOnce(() => {
+      return yamlConfig;
+    });
+
+    render(
```

Comment:
@karesti Need your help on this error.

Got this when running jest test on CacheConfiguration.test.tsx

![Screenshot from 2022-02-02 16-30-48](https://user-images.githubusercontent.com/97534722/152142404-3a2a18cd-9236-466c-9119-a0a6e35f89ef.png)